crates/holo_hash/Cargo.toml

```diff
@@ -23,6 +23,7 @@ derive_more = { version = "0.99", optional = true }
 fixt = { version = "^0.2.0", path = "../fixt", optional = true }
 futures = { version = "0.3", optional = true }
 holochain_serialized_bytes = { version = "=0.0.51", optional = true }
+holochain_util = { version = "^0.2.0", path = "../holochain_util", default-features = false }
```

Comment:
holo_hash is [used in wasms](https://github.com/holochain/holochain/blob/806265253d927bd1506c23a5485df21585811042/crates/holochain_integrity_types/Cargo.toml#L13)... If you want to include here, you're going to need to clean up the features in the holochain_util crate itself. For example the [ffs and tokio_helper modules](https://github.com/holochain/holochain/blob/806265253d927bd1506c23a5485df21585811042/crates/holochain_util/src/lib.rs#L3-L4) are not respecting the relevant feature flippers.

Reply:
oh, haha, you did that, I just had to get down there :+1: